packages/account/src/test-utils/launch-custom-provider-and-get-wallets.test.ts

```diff
@@ -0,0 +1,190 @@
+import { BaseAssetId } from '@fuel-ts/address/configs';
+import { safeExec } from '@fuel-ts/errors/test-utils';
+import { toHex, toNumber } from '@fuel-ts/math';
+
+import { Provider, sleepUntilTrue } from '../providers';
+import { Signer } from '../signer';
+import { WalletUnlocked } from '../wallet';
+
+import { AssetId } from './asset-id';
+import { launchCustomProviderAndGetWallets } from './launch-custom-provider-and-get-wallets';
+import { TestMessage } from './test-message';
+import { urlIsLive } from './utils';
+import { WalletConfig } from './wallet-config';
+
+/**
+ * @group node
```

Comment:
Can and should these also be browser tests? I think this can be applied to all the tests you've added.

Reply:
They can't be browser tests because `launchNode` depends on node-specific libraries (e.g. `child_process`). I was thinking about browser tests and I think the only way for this to work with them is to spin up a server with a "generateNode" endpoint that'd return the url onto which the browser can connect for that specific test. It's way out of the scope of this PR, though :smile:

Reply:
Ah of course, yeah agreed that's a future nice to have